<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfEditor</class>
 <widget class="QMainWindow" name="ConfEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>495</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ConfEditor</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="confTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="protoCombo">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>all</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>tcp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>udp</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="actionCombo">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>crypt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>drop</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>proto:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>action:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>370</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="cryptGroup">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>311</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>crypt parameters:</string>
    </property>
    <widget class="QLabel" name="formatLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>key format:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="keyEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="methodLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>method:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="formatCombo">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>104</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>plain</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>base64</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="keyLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>key:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="methodCombo">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>121</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>aes-128-gcm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>aes-256-gcm</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>370</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSaveTo">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
